### PR TITLE
fix: create sub pages for whitepaper and economics

### PR DESCRIPTION
**Pull request by forbesus**
ref #404

### CODE REVIEW

File: src/pages/whitepaper.astro [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<Layout title="Redirecting to Whitepaper">
+  <head>
+    <meta http-equiv="refresh" content={`0; url=${redirectUrl}`} />
+  </head>
+  <div class="relative overflow-hidden">
+    <div class="container text-center">
+      <div class="border-b border-[#d9d9d9] pb-[40px] pt-5 lg:pt-[60px] text-center">
+        <h1 class="text-xl font-bold md:text-3xl lg:text-4xl leading-tight">
+          Redirecting to Whitepaper
```
Comment (hiroyukikumazawa) [inline src/pages/whitepaper.astro]: @forbesus please update this to `Redirecting to Akash Whitepaper.`

File: src/pages/whitepaper.astro [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+import Layout from "@/layouts/layout.astro";
+const redirectUrl = "https://ipfs.io/ipfs/QmVwsi5kTrg7UcUEGi5UfdheVLBWoHjze2pHy4tLqYvLYv";
+---
+
+<Layout title="Redirecting to Whitepaper">
```
Comment (hiroyukikumazawa) [inline src/pages/whitepaper.astro]: @forbesus this too

**Comment by HoomanDgtl** [comment]
under review

**Comment by HoomanDgtl** [comment]
@brewsterdrinkwater @chainzero : this is approved and ready to be merged..

**Pull request merged**